### ISSUE TITLE
Resume pipeline after -os 

### ISSUE THREAD

**Issue by swuyts** [davidemms/OrthoFinder#90]
Hi there,

We've ran an OrthoFinder analysis wtih the -os option.

However, with the new update, we'd like to give the rest of the pipeline a chance so I've tried running:

`orthofinder -fg $OrthoFinderDir -M msa`

This always gives the error:
`ERROR: cannot find files from OrthoFinder run in directory:
   ~/orthofinder/Results_Jun26/
or
   ~/orthofinder/Results_Jun26/WorkingDirectory/
`

As stated in the manual, this is the dir that contains the result files "Orthogroups.csv". 

Is it possible that by using the -os option that we actually lost the ability to continue the pipeline? Is there a workaround (e.g. add a new species?)

Many thanks,
Sander

**Comment by davidemms** [comment]
Hi Sander

I've just checked that the workflow you describe works fine with the example dataset:
1. 
`python orthofinder.py -f ~/vtemp/ExampleDataset -M msa -os`

> OrthoFinder version 1.1.9 Copyright (C) 2014 David Emms
> ...
> etc
> ...
> Orthogroups have been written to tab-delimited files:
>    /home/david/vtemp/ExampleDataset/Results_Jun29/Orthogroups.csv
>    /home/david/vtemp/ExampleDataset/Results_Jun29/Orthogroups.txt (OrthoMCL format)
>    /home/david/vtemp/ExampleDataset/Results_Jun29/Orthogroups_UnassignedGenes.csv
> 
> Sequences for orthogroups:
>    /home/david/vtemp/ExampleDataset/Results_Jun29/Orthologues_Jun29/Sequences
> 
> Orthogroup statistics:
>    Statistics_PerSpecies.csv   Statistics_Overall.csv   Orthogroups_SpeciesOverlaps.csv
> 
> OrthoFinder assigned 1938 genes (70.9% of total) to 536 orthogroups. Fifty percent of all genes were in orthogroups 
> with 4 or more genes (G50 was 4) and were contained in the largest 299 orthogroups (O50 was 299). There were 278 
> orthogroups with all species present and 254 of these consisted entirely of single-copy genes.
> 
> When publishing work that uses OrthoFinder please cite:
>     D.M. Emms & S. Kelly (2015), OrthoFinder: solving fundamental biases in whole genome comparisons
>     dramatically improves orthogroup inference accuracy, Genome Biology 16:157.

2.
`python orthofinder.py -fg ~/vtemp/ExampleDataset/Results_Jun29 -M msa`

> OrthoFinder version 1.1.9 Copyright (C) 2014 David Emms
> 
>     This program comes with ABSOLUTELY NO WARRANTY.
>     This is free software, and you are welcome to redistribute it under certain conditions.
>     For details please see the License.md that came with this software.
> 
> 16 thread(s) for highly parallel tasks (BLAST searches etc.)
> 1 thread(s) for OrthoFinder algorithm
> 
> Using orthogroups in file:
>     /home/david/vtemp/ExampleDataset/Results_Jun29/Orthogroups.txt
> and corresponding clusters file:
>     /home/david/vtemp/ExampleDataset/Results_Jun29/WorkingDirectory/clusters_OrthoFinder_v1.1.9_I1.5.txt_id_pairs.txt
> ...
> etc
> ...

and that worked fine (the full output is in the log file: [TwoSteps.txt](https://github.com/davidemms/OrthoFinder/files/1112401/TwoSteps.txt)

Could you just check that that also works for you with the Example Dataset and that you're carrying out the same steps with the dataset you want to analyse? If this doesn't work you you call 'ls' on the ~/orthofinder/Results_Jun26/WorkingDirectory/ directory and post the list of files that it contains?

Thanks
David




.

**Comment by swuyts** [comment]
David, 

I've ran your analysis and indeed everything worked fine with ExampleDatasets. 

So indeed I've dug a little bit deeper, and something went wrong in transferring files from one place to another.  The original Results folder has around 67,499 files wile mine only had 40,853. I transferred everything again, and now it's running smoothly.  

Apologies for not thoroughly checking that first, should have done that prior to posting. 

Cheers,
Sander
